src/main/java/com/wire/bots/hold/Service.java

```diff
@@ -139,6 +158,10 @@ public void run(Config config, Environment environment) {
         CollectorRegistry.defaultRegistry.register(new DropwizardExports(metrics));
 
         environment.getApplicationContext().addServlet(MetricsServlet.class, "/metrics");
+
+        // todo here
+        // String res = loginClient.getBackendConfiguration();
+        // handle res ^
```

Comment:
This will be replace by the implementation of `FallbackDomainFetcher` from https://github.com/wireapp/legalhold/pull/27

Reply:
Yes, this is only present because I started coding this based on your feature branch, I hoped to get less merge conflict when we merge both PRs, we will see